backend/routes/getters.js

```diff
@@ -16,6 +41,7 @@ router.get("/stats/:companyTick", async (req, res) => {
   res.status(200).json(result);
 });
 
+// explore page, will be filled with better suggestions later
```

Comment:
Either remove this comment completely or use "TODO" format:
`// TODO: fill explore page with suggestions`